packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1769,7 +1769,7 @@ abstract class RenderBox extends RenderObject {
         throw new FlutterError(
           '$runtimeType does not implement applyPaintTransform.\n'
           'The following $runtimeType object:\n'
-          '  ${this.toStringShallow()}\n'
+          '  ${toStringShallow()}\n'
```

Comment:
Not sure how I feel about this one. I'd intentionally put in the "this" to match the "child" case below.
I guess I'll live with it.